docs/pages/management/operations/upgrading.mdx

```diff
@@ -55,19 +54,19 @@ When upgrading a single Teleport cluster:
 
 <Notice type="tip">
 Teleport requires upgrading the Teleport Cluster from one major version to another in sequence.
-That means if you are currently on version 8.X you must upgrade to the latest version 9 before
-going to version 10. To upgrade in sequence repeat upgrading the Teleport Cluster, confirming
+If you are currently on version 8.x, you must upgrade to the latest version 9.x before
```

Comment:
While we're here, if you want to update the example to more recent versions (like 11, 12, and 13), that would be fine :-)